snappy/security.go

```diff
@@ -20,207 +20,848 @@
 package snappy
```

Comment:
I had a similar idea when looking at this. I think its a good idea however right now we use packageYaml all over the place in security.go so that would require some refactor to not use that. I can do a folllowup branch with this refactor if you want and we can decide what to do with it.